docs/discovery.md

```diff
@@ -1,18 +1,50 @@
-
-## Dynamic Discovery
+# Dynamic Discovery
 The device service supports [dynamic discovery](https://docs.edgexfoundry.org/2.1/microservices/device/Ch-DeviceServices/#dynamic-provisioning).
 During dynamic discovery, the device service scans all connected USB devices and sends the discovered cameras to Core Metadata.
 The device name of the camera discovered by the device service is comprised of Card Name and Serial Number, and the characters colon, space and dot will be replaced with underscores as they are invalid characters for device names in EdgeX.
 Take the camera Logitech C270 as an example, it's Card Name is "C270 HD WEBCAM" and the Serial Number is "B1CF0E50" hence the device name - "C270_HD_WEBCAM-B1CF0E50".
 
 > NOTE: Card Name and Serial number are used by the device service to uniquely identify a camera. Some manufactures, however, may not support unique serial numbers for their cameras. Please check with your camera manufacturer.
 
-### Enable the Dynamic Discovery function
-Dynamic discovery is enabled by default to save computing resources.
-If you want the device service to run the discovery periodically, enable it and set a desired interval.
-The interval value must be a [Go duration](https://pkg.go.dev/time#ParseDuration).
+## Dynamic Discovery function
+Dynamic discovery is enabled by default to make setup easier.
+If you wish to not run discovery periodically, disable it by changing the `Enabled` option to `false` as shown below.
+
+
+### Disable discovery
 
-[Option 1] Enable from the [configuration.toml](../cmd/res/configuration.toml)
+[Option 1] Disable discovery from the [configuration.toml](../cmd/res/configuration.toml) file.
+```yaml
+[Device] 
+...
+    [Device.Discovery]
+    Enabled = false
+    Interval = "1h"
+```
+
+[Option 2] Disable discovery from environment variables. 
+```shell
+export DEVICE_DISCOVERY_ENABLED=false
+export DEVICE_DISCOVERY_INTERVAL=1h
+```
+
+[Option 3] Disable discovery from a curl command.
+
+//TODO: add discovery command
+To stop the usb camera from live streaming, use the following command:
+
+Query parameter:
+- `device name`: The name of the camera
+
+For example:
+```shell
+curl -X PUT -d '{
+    "StopStreaming": "true"
+}' http://localhost:59882/api/v2/device/name/<device name>/Discovery
```

Comment:
I don't think this is supposed to be in here

Reply:
its a TODO for me.. I'm going to remove it.

Reply:
Sounds good